src/app/WriteClient.h

```diff
@@ -119,7 +132,62 @@ class WriteClient : public Messaging::ExchangeDelegate
     System::PacketBufferTLVWriter mMessageWriter;
     WriteRequest::Builder mWriteRequestBuilder;
     uint8_t mAttributeStatusIndex = 0;
-    intptr_t mAppIdentifier       = 0;
+    uint64_t mAppIdentifier       = 0;
+};
+
+class WriteClientHandle
+{
+public:
+    /**
+     * Construct an empty PacketBufferHandle.
```

Comment:
All of these copy/pasted comments should obviously be fixed....

Reply:
Ah, yes, thanks.